keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -1,20 +1,28 @@
 package com.netflix.spinnaker.keel.api.artifacts
 
-enum class ArtifactType {
-  deb, docker;
-
-  override fun toString() = name
-}
+typealias ArtifactType = String
 
+/**
+ * The release status of an artifact. This may not necessarily be applicable to all
+ * [DeliveryArtifact] sub-classes.
+ */
 enum class ArtifactStatus {
```

Comment:
Can those statuses by applicable for other types other than `debians`?

Reply:
I don't know, so I'm trying to future-proof the interfaces in case we do.